odoo/addons/base/ir/ir_qweb/assetsbundle.py

```diff
@@ -480,7 +480,7 @@ def _fetch_content(self):
                 with open(self._filename, 'rb') as fp:
                     return fp.read().decode('utf-8')
             else:
-                return self._ir_attach['datas'].decode('base64')
+                return self._ir_attach['datas'].decode('base64').decode('utf-8')
```

Comment:
Seem good, but, check and/or add test if it's ok for javascript https://github.com/odoo/odoo/pull/14630/files/b9200ac5aab236d5672fb1cccc02cd6f094df0e5?diff=unified#diff-00fd70a5097b17dd775a9baceffc5a8aL506 with special char